florist/api/server.py

```diff
@@ -0,0 +1,118 @@
+"""FLorist server FastAPI endpoints."""
+import logging
+from typing import List
+
+import requests
+from fastapi import FastAPI, Form
+from fastapi.responses import JSONResponse
+from typing_extensions import Annotated
+
+from florist.api.monitoring.metrics import wait_for_metric
+from florist.api.servers.common import ClientInfo, ClientInfoParseError, Model
+from florist.api.servers.launch import launch_local_server
+
+
+app = FastAPI()
+LOGGER = logging.getLogger("uvicorn.error")
```

Comment:
Is it better to define our own logger as opposed to using the uvicorn.error one? I was taking a look at how [flwr does it](https://github.com/adap/flower/blob/main/src/py/flwr/common/logger.py) and it seems pretty simple to define a library specific logger. I could be missing something though so let me know what you think :)

Reply:
Sounds good!